infra/test/setup/versions.tf

```diff
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.48"
+      version = "~> 5.22"
     }
     google-beta = {
       source  = "hashicorp/google-beta"
-      version = "~> 4.48"
+      version = "~> 5.22"
```

Comment:
The integration tests were failing during project setup:
```
hashicorp/google-beta: no available releases match the given constraints >=
3.43.0, >= 4.11.0, ~> 4.48, >= 5.22.0, < 6.0.0
```
This is because the dependency being bumps requires provider >= 5.22.0.
See https://github.com/terraform-google-modules/terraform-google-project-factory/blob/ac038594dfbb78c883003e119002fa300f89072f/versions.tf#L19.